### PR TITLE
adds a colab tutorial

### PR DESCRIPTION
**Pull request by wyli**
Fixes https://github.com/Project-MONAI/MONAI/issues/391

### Description
this is colab example, I'll improve the content based on feedback

### Status
**Ready**

### Types of changes

- [x] Docstrings/Documentation updated

### CODE REVIEW

File: examples/README.md [hunk 1]
```diff
@@ -17,6 +17,7 @@ pip install -r https://raw.githubusercontent.com/Project-MONAI/MONAI/master/requ
 ```
 
 ### 2. List of examples
+#### [A brief guide to MONAI for PyTorch users (Colab)](https://colab.research.google.com/drive/1boqy7ENpKrqaJoxFlbHIBnIODAs1Ih1T)
```
Comment (Nic-Ma) [inline examples/README.md:20]: 1. Suggest also add this link to our main README directly in "Quick Start" guide, what do you think?
image
2. Suggest also adding the previous `MedNIST Colab notebook` to this README and the quick start of main README.

Comment (wyli) [reply examples/README.md]: sure I changed the link and I'll update the notebook

Comment (Nic-Ma) [reply examples/README.md]: Cool, let's merge this PR when your Colab notebook is finalized.
Thanks.

**Comment by wyli** [comment]
> Looks good to me now.
> Let's wait to merge this PR when your Colab notebook is finalized.
> 
> Thanks.

I give you edit permission to both notebooks, could you carry on maintaining the tutorials?

**Pull request merged**